scripts/release.star

```diff
@@ -71,7 +71,7 @@ def get_steps(edition, ver_mode):
     steps = [
         codespell_step(),
         shellcheck_step(),
-        test_backend_step(edition=edition),
+        test_backend_step(edition=edition, publish=should_publish, tries=5),
```

Comment:
As I commented elsewhere in the PR, `tries` should be set to None if `should_publish` is true.